### PR TITLE
Updated method Zephir\ClassDefinition::buildFromReflection

### PR DESCRIPTION
**Pull request by dreamsxin**
When `config.json` set `extension-name` will exec:
```shell
configure --enable-{$extension-name}
```

### CODE REVIEW

File: Library/Compiler.php [hunk 1]
```diff
@@ -1197,6 +1197,10 @@ public function compile(CommandInterface $command, $development = false)
          * Get global namespace
          */
         $namespace = str_replace('\\', '_', $this->checkDirectory());
+		$extensionName  = $this->config->get('extension-name');
+		if (empty($extensionName) || !is_string($extensionName)) {
+			$extensionName = $namespace;
+		}
```
Comment (sergeyklay) [inline Library/Compiler.php]: > Spaces must be used to indent lines; tabs are not allowed

**Review by sergeyklay — CHANGES_REQUESTED** [review]

File: Library/Compiler.php (hunk 1 above)
Comment (dreamsxin) [reply Library/Compiler.php]: Oh, I will correct it, ths.

**Comment by dreamsxin** [comment]
@sergeyklay It'sfix.

**Comment by sergeyklay** [comment]
Thanks

**Pull request merged**